src/hooks/useEditModeInformation.ts

```diff
@@ -0,0 +1,11 @@
+import { create } from 'zustand';
+
+interface EditModeInformationStore {
+  isEditModeDisabled: boolean;
```

Comment:
rename to `editModeEnabled` to avoid negations is boolean naming 

https://www.serendipidata.com/posts/naming-guidelines-for-boolean-variables

Reply:
Good suggestion. Perhaps we should add such stuff to the contribution guidelines?